application/src/View/Helper/SearchFilters.php

```diff
@@ -48,7 +48,7 @@ public function __invoke($partialName = null, array $query = null)
                 switch ($key) {
                     // Fulltext
                     case 'fulltext_search':
-                        $filterLabel = $translate('Search full-text');
+                        $filterLabel = $translate('Search full-text'); // @translate
```

Comment:
`$translate` calls are detected by xgettext, so they don't need the `@translate` comment.

Generally the comment is only needed where the string literal is not used directly in a call to `$translate` or `$this->translate`.

Reply:
Same comment applies to all the changes in this file, I believe